kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/ResourceListTable.vue

```diff
@@ -3,19 +3,14 @@
   <core-table>
```

Comment:
Handle empty state.

Reply:
this one is handled: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/index.vue#L41